src/Microsoft.AspNetCore.Routing/DefaultLinkGenerator.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Text.Encodings.Web;
 using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing.Features;
```

Comment:
hmm, where did this namespace come from? I guess the other PR is the place to discuss.